donut/modules/groups/utils/insert_permissions.py

```diff
@@ -65,14 +65,14 @@
         # View Arcfeedback emails, Surveys
         True: [
                 arc_permissions.SUMMARY,
-                arc_permissions.TOGGLE_READ,
+                arc_permissions.TOGGLE_RESOLVED,
                 arc_permissions.ADD_REMOVE_EMAIL,
                 arc_permissions.VIEW_EMAILS,
                 voting_permissions.SURVEYS],
         # View Arcfeedback summary, Edit Arcfeedback, View Arcfeedback emails,
         # Surveys
         False: [arc_permissions.SUMMARY,
-                arc_permissions.TOGGLE_READ,
+                arc_permissions.TOGGLE_RESOLVED,
```

Comment:
Did you change the name of these permissions in the DB? They should also probably be changed on the wiki page: https://github.com/ASCIT/donut/wiki/Managing-the-production-server#initial-configuration